test/support/test_helpers.ex

```diff
@@ -0,0 +1,11 @@
+defmodule RecipeBox.TestHelpers do
+  alias RecipeBox.Cuisine
+
+  def cuisine_fixture(attrs \\ %{}) do
```

Comment:
It doesn't look like this helper is being called anywhere right now. I'd say let's pull it out of this branch so it can be included alongside the code that calls it in the future.

Reply:
Ah, thanks for catching that. I've been working on two separate branches, and did a little cherry picking, so that must have been accidentally included.